roles/azure_manage_postgresql/README.md

```diff
@@ -29,7 +29,13 @@ Role Variables
 * **azure_manage_postgresql_postgresql_enforce_ssl**: Enable SSL enforcement. Default: False
 * **azure_manage_postgresql_postgresql_storage_autogrow**: Enable storage autogrow. Default: False
 * **azure_manage_postgresql_postgresql_admin_username**: The administrator's login name of a server. Can only be specified when the server is being created (and is required for creation).
-* **azure_manage_postgresql_postgresql_admin_password**: The password of the administrator login. When this is not defined, the role will generated a password that can be read later in the variable name.
+* **azure_manage_postgresql_postgresql_admin_password**: The password of the administrator login. When this is not defined, the role will generated a password that can be read later in the variable name.  
+**The supplied password must be between 6-72 characters long and must satisfy at least 3 of password complexity requirements from the following**:
```

Comment:
I don't think this should be bold, but not critical

Reply:
Bold removed